cogs/induct.py

```diff
@@ -201,16 +201,16 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
             return
 
         intro_channel: discord.TextChannel | None = discord.utils.get(
-            self.bot.main_guild.text_channels, 
+            self.bot.main_guild.text_channels,
             name="introductions",
         )
 
         if intro_channel:
-            message: discord.Message
-            for message in await intro_channel.history(limit=30).flatten():
-                if m.author.id == induction_member.id:
-                    await m.add_reaction(":wave:")
-                    await m.add_reaction(":TeX:")
+            msg: discord.Message
+            for msg in await intro_channel.history(limit=30).flatten():
```

Comment:
Variable names should not be shortened abbreviations. Use `message` here.
If this makes line 210 too long, a `# noqa: E501` declaration can be used, as for-loops cannot be split onto multiple lines.

Reply:
can't use message cos it's reassigned later

Reply:
Use `recent_message` then👍🏻